src/tfvc/tfvc-extension.ts

```diff
@@ -32,6 +32,15 @@ export class TfvcExtension  {
         this._manager = manager;
     }
 
+    public async AssociateWorkItems(): Promise<void> {
+        this.displayErrors(async () => {
```

Comment:
Let's add Telemetry.SendEvent("tfvc.associateworkitems") so we can have some indication on how much the command is used.